extension/content/firebug/lib/http.js

```diff
@@ -92,11 +92,22 @@ Http.readPostTextFromPage = function(url, context)
      }
 };
 
-Http.getResource = function(aURL, ignoreMissing)
+Http.getResource = function(aURL, ignoreMissing, doc)
 {
```

Comment:
I don't see any place where you'd pass a document to the function so, why a new argument?

Honza